### PR TITLE
buildship updatesite url has been changed.

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by jtydhr88** [comment]
Hi Greg, the native fixed be included in this pr

File: build/com.liferay.ide-repository/hidden.features.product [hunk 1]
```diff
@@ -49,6 +49,7 @@
       <feature id="org.eclipse.sapphire.ui.swt.xml.editor" />
       <feature id="org.eclipse.wst.xml.search.core-feature" />
       <feature id="org.sonatype.m2e.mavenarchiver.feature" />
+	  <feature id="org.sonatype.tycho.m2e.feature" />
```
Comment (gamerson) [inline build/com.liferay.ide-repository/hidden.features.product]: Check formatting on this line, seems like an extra tab.

File: build/parent/pom.xml [hunk 2]
```diff
@@ -44,7 +44,8 @@
         <tern-site>http://oss.opensagres.fr/tern.repository/0.10.0/</tern-site>
         <webresources-site>http://oss.opensagres.fr/eclipse-wtp-webresources/0.9.0-SNAPSHOT/</webresources-site>
         <bndtools-site>https://bndtools.ci.cloudbees.com/job/bndtools.master/lastSuccessfulBuild/artifact/build/generated/p2/</bndtools-site>
-        <gradle-site>http://download.eclipse.org/buildship/updates/snapshots/1.0</gradle-site>
+        <gradle-site>http://download.eclipse.org/buildship/updates/e45/snapshots/1.0/</gradle-site>
+		<tycho-m2e-site>http://repo1.maven.org/maven2/.m2e/connectors/m2eclipse-tycho/0.8.0/N/LATEST/</tycho-m2e-site>
```
Comment (gamerson) [inline build/parent/pom.xml]: Check formatting on this line, seems like an extra tab.

**Comment by gamerson** [comment]
Closing see comments.

**Pull request closed without merging**